### ISSUE TITLE
Digitize Travel Requests and Reimbursements

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#663]
The finance manages staff travel requests and reimbursements through a combination of Excel, PDFs, and lots of printed paper. It requires nearly a full FTE to manage. Everyone is unhappy. No doubt there are out of the box solutions, but like so much of our business, I suspect a custom app is better suited to address the information capture and workflows our bureaucracy requires.

**Comment by dianamartin** [comment]
@amenity this sounds like a fun project.. :)

**Comment by amenity** [comment]
@dianamartin @stephaniemoorer - this is a cool idea, but considering that Finance didn't make the request to begin with, what do you think about closing it out?

**Comment by dianamartin** [comment]
@amenity cc @stephaniemoorer Finance has some items they plan to give us for enhancements, we can bring this up when we're prioritizing with them to see if this is even a valid idea for them. If not, we can close it.

**Comment by amenity** [comment]
@dianamartin @stephaniemoorer - any news? 🍿

**Comment by dianamartin** [comment]
@amenity cc @stephaniemoorer We have a meeting with them this morning about another enhancement, we can maybe bring it up then

**Comment by maccallump** [comment]
@stephaniemoorer can you tell me who the stakeholder is for this request?  
At the FY26-27 Budget Kickoff there was a discussion about additional scrutiny on travel.  It would be great to talk about the needs.

**Comment by dianamartin** [comment]
@maccallump I think this was John, back when he was a supervisor. I imagine other TPW Supervisors would probably agree with streamlining this process a bit.

**Comment by johnclary** [comment]
yes, I think this one came out of our earliest business needs assessment. Possibly even an Anthony Segura idea.
